Member-Travel-Fund.md

```diff
@@ -1,41 +1,61 @@
 ## Purpose
 
 To establish and administer a fund for members of the Node.js
-Foundation to travel and spread knowledge about and support the foundation
-and the Node.js Foundation projects.
+Foundation to travel, spread knowledge about, and support the Node.js 
+Foundation projects.
 
 ## Restrictions
 
 * Candidates must be Individual Members of the Node.js Foundation.
 
+### Travel Fund Administrators
+
+Members of both the TSC and CommComm are responsible for approving fund requests
+and communicating approval/denial through the PR filed by the member requesting. 
+
+Travel fund admins will consider each proposal tagged for their respective group 
+in its next regular meeting. Requests tagged for both committees must be coordinated 
+and communicated together by the Chair of each. See considerations below to set 
+yourself up for success to be approved.
+
+Linux Foundation Accounts Payable and the Node.js community manager are responsible for 
+verifying the requester is an Individual Member of the Node.js Foundation and dispersing funds,
+as required to receive funds(see Reimbursement below).
+
```

Comment:
It may be worthwhile explicitly stating here that only Individual Members are eligible to receive funds. I know it's implied, but it's worth repeating.